packages/edit-site/src/components/editor/global-styles-provider.js

```diff
@@ -30,12 +31,17 @@ const GlobalStylesContext = createContext( {
 
 export const useGlobalStylesContext = () => useContext( GlobalStylesContext );
 
+const useGlobalStylesEntityContent = () => {
+	return useEntityProp( 'postType', 'wp_global_styles', 'content' );
+};
+
+export const useGlobalStylesReset = () => {
+	const [ , setContent ] = useGlobalStylesEntityContent();
+	return useCallback( () => setContent( '' ), [ setContent ] );
```

Comment:
Not a big deal, perhaps setting it to an empty object would make more sense?

```suggestion
	return useCallback( () => setContent( '{}' ), [ setContent ] );
```

Reply:
Actually, this needs to be `{}`, otherwise the PHP code that de-serializes this will report a syntax error.